Integrations/integration-EWSMailSender.yml

```diff
@@ -42,34 +43,46 @@ configuration:
   defaultvalue: "false"
   type: 8
   required: false
+- display: Has impersonation rights.
+  name: impersonation
```

Comment:
why do we need impersonation for send ?

Reply:
Sometimes only impersonation will be possible (in some exchange config)